### DISCUSSION TITLE
Minecraft Mod 3D Model textures won't show

### DISCUSSION THREAD

**Discussion by DiaMondTH** [community/community#144526] [Programming Help]
### 3D Model textures won't show

Question

### Body

Anyone know how to help me with **minecraft 1.21 fabric 0.16.9** mod? I've been having problems with this for days now, my ** 3D Model textures won't show** up. I'm really frustrated. This mod is a project I have to submit to my **teacher** in a few months (can someone help me please)

![ภาพหน้าจอ (1)](https://github.com/user-attachments/assets/797ae998-9d23-4e21-aa03-daafe73cf650)
![ภาพหน้าจอ (2)](https://github.com/user-attachments/assets/3b513689-7da1-41ec-8d21-090f6d9918c2)
![ภาพหน้าจอ (3)](https://github.com/user-attachments/assets/73234fc4-f2d5-4d5c-96b1-43c0be8c374c)

**Comment by themodderg** [comment]
Hello I think I know what is happening.

You are setting in your json corretly the **breaking particle** correctly, but you are not setting up the texture for the model itself.

### How to correctly setup the texture for the model

"textures": {
    "0": "companions:block/copper_coin",
    "particle": "companions:block/copper_coin"
  }

this bit is from a mod I'm making with a friend, check how we set the texture indexed on zero (thats the one you want for the model) **and** the texture for the breaking particle. Once you add both it should work, you may also need to add it on the 1 index instead.

Have a good day.
